modules/RTC/JitsiLocalTrack.js

```diff
@@ -69,6 +74,19 @@ export default class JitsiLocalTrack extends JitsiTrack {
             mediaType,
             videoType);
 
+        this.effects = effects;
+
+        this._videoBlurEnabled = this.effects.indexOf('blur') !== -1
+            && mediaType === MediaType.VIDEO;
+
+        this._jitsiStreamBlurEffect = null;
```

Comment:
You can leave this set to `undefined`, isn't it? Isn't that going to be better? WDYT?

Reply:
Agree, undefined is better.